src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
```

Comment:
Don't add empty lines between a parent and a child components
```suggestion
    <form
```